src/abstractdataframe/sort.jl

```diff
@@ -341,10 +341,15 @@ If `rev` is `true`, reverse sorting is performed. To enable reverse sorting only
 for some columns, pass `order(c, rev=true)` in `cols`, with `c` the
 corresponding column index (see example below).
 
+Since having repeated elements makes multiple sorting orders valid, if `checkunique`
+is `true` some basic uniqueness checks are made. If duplicate elements are 
```

Comment:
why do you say "basic"? They are correct when they work. Rather please specify when this kwarg is allowed.

Reply:
Here is the reworked version:
```
... corresponding column index (see example below).

Since having repeated elements makes multiple sorting orders valid, the
`checkunique` keyword allows for the situation to be caught. If `checkunique` is
`true` and duplicate elements are found an `ArgumentError` will be thrown. The
use of the `checkunique` keyword is only supported when neither the `by` nor
`lt` keywords are being used, as their application can create duplicate items
inadvertently. Similarly, the use of `order(...)` clauses that specify either
`by` or `lt` are not supported, but specifying `rev` by itself is allowed.

The `by` keyword allows providing a function that will be applied to each
cell before comparison; the `lt` keyword allows providing a custom "less
than" function. If both `by` and `lt` are specified, the `lt` function is
applied to the result of the `by` function.

All the keyword arguments can be either a single value, which is applied to ...
```

Is it more helpful?

Reply:
Yes. Except the other comment I left that `All the keyword arguments can be either` in the last line is incorrect (since `checkunique`) should be excluded.